### PR TITLE
fix(sdk): best_of in completions was an int and should be a string

### LINKED ISSUE #819: bug(api): Fix legacy `completions` API endpoint
### Describe what should be investigated or refactored
The legacy `/openai/v1/completions` endpoint is broken and returns an `500` Internal Server Error. This endpoint should be fixed to operate according to the legacy endpoint specification.

### Links to any relevant code
- [Legacy `completions` endpoints](https://platform.openai.com/docs/api-reference/completions/create)
- [New `chat/completions` endpoint](https://platform.openai.com/docs/api-reference/chat/create)

### Additional context
The `Continue.dev` plug-in seems to require this endpoint, so we should continue to maintain it until we learn otherwise.

### PR DESCRIPTION
**Pull request by lemonprogis**
## Description

the `best_of` in completion.proto hasn't been changed to a string like in others. Changed that and ran the code gen. 

### BREAKING CHANGES
N/A

## Related Issue

Fixes #819 

## Checklist before merging

- [ ] Tests, documentation, ADR added or updated as needed
- [ ] Followed the [Contributor Guide Steps](https://github.com/defenseunicorns/leapfrogai/blob/main/.github/CONTRIBUTING.md)

### CODE REVIEW

File: src/leapfrogai_sdk/audio/audio_pb2.py
Comment (gphorvath) [inline src/leapfrogai_sdk/audio/audio_pb2.py:1]: It appears the lint is change tab size from 4 to 2 spaces. 4 should be the default for Python

File: src/leapfrogai_sdk/audio/audio_pb2.pyi
Comment (gphorvath) [inline src/leapfrogai_sdk/audio/audio_pb2.pyi:1]: In this file it seems like somebody's linter config is undoing the line length limit and removing trailing whitespace.

File: src/leapfrogai_sdk/audio/audio_pb2_grpc.py
Comment (gphorvath) [inline src/leapfrogai_sdk/audio/audio_pb2_grpc.py:1]: Double quotes changing to single quotes - check linter config.

**Review by gphorvath — CHANGES_REQUESTED** [review]
I would like to clear up the linting issues before merging. There is a lot of this that is a no-op diff in generated proto code, due to some difference in linting config. I am also curious why our linting workflow didn't catch this.

File: src/leapfrogai_sdk/proto/leapfrogai_sdk/completion/completion.proto
[the remark this answers is not in the document]
Comment (lemonprogis) [reply src/leapfrogai_sdk/proto/leapfrogai_sdk/completion/completion.proto:1]: Interesting, I'll take a look but I ran 'make gen-python'. Should there be more granularity added where one would only generate a specific module?

File: src/leapfrogai_sdk/audio/audio_pb2.py
Comment (lemonprogis) [reply src/leapfrogai_sdk/audio/audio_pb2.py:1]: You are correct, I didn't have pre-commits setup on the repo yet. I reverted the initial commit, setup pre-commit, and then made the changes.

File: src/leapfrogai_sdk/proto/leapfrogai_sdk/completion/completion.proto
[the remark this answers is not in the document]
Comment (gphorvath) [reply src/leapfrogai_sdk/proto/leapfrogai_sdk/completion/completion.proto:1]: I'm not sure, but with the linter tweaks this looks better!

**Review by gphorvath — APPROVED** [review]

**Review by jalling97 — APPROVED** [review]
Thanks for pushing these fixes!

**Review by justinthelaw — APPROVED** [review]
@lemonprogis feel free to smash that merge button!

**Pull request merged**